src/Core/Silk.NET.SilkTouch/NativeContextOverrides/PInvokeNativeContextOverride.cs

```diff
@@ -8,19 +8,20 @@
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+using static Silk.NET.SilkTouch.NameGenerator;
 
 namespace Silk.NET.SilkTouch.NativeContextOverrides
 {
     public sealed class PInvokeNativeContextOverride : INativeContextOverride
     {
         /// <inheritdoc />
-        public TypeDeclarationSyntax Type(string name, string lib, EntryPoint[] entrypoints)
+        public TypeDeclarationSyntax Type(string name, string lib, EntryPoint[] entrypoints, Compilation comp)
         {
```

Comment:
if we only pass the compilation to get this bool, why not just pass the bool?

Reply:
Changed to use a new context structure which does not have a compilation within it.